numba/tests/test_llvm_pass_timings.py

```diff
@@ -42,8 +42,11 @@ def foo(n):
                 c += i
             return c
 
-        with override_config('LLVM_PASS_TIMINGS', True):
-            foo(10)
+        # FIXME: Pass timings is not fully supported right now in NewPassManager
```

Comment:
I believe some downstream users of Numba are interested in pass timings. What is the route to full support for them?

Reply:
With NPM I couldn't find a way to redirect the pass timer output to a filestream(like how it works with the legacy PM) from the std::error stream. I suppose it's an implementation issue and can be fixed.